### PR TITLE
Issue#23 setup react router

### LINKED ISSUE #23: Setup react-router

### PR DESCRIPTION
**Pull request by husseinTalal2**
installing react-router and linking navbar and footer links to the router closes #23

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,13 +3,16 @@ import 'bootstrap/dist/css/bootstrap.min.css';
...
+import AboutPage from './containers/AboutPage/About';
 function App() {
   return (
-    <div>  
+    <div>
       <Nav />
-      <HomePage />
+      <Switch>
+        <Route exact path="/" component={HomePage} />
+        <Route path="/AboutPage" component={AboutPage} />
```
Comment (MustafaNawzat) [inline src/App.js:14]: path="/AboutPage" >> path="/about"

the path should be all lower case

File: src/components/AboutSection/AboutSection.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```
Comment (MustafaNawzat) [inline src/components/AboutSection/AboutSection.js:1]: why this file is showing here?

File: src/App.js (hunk 1 above)
Comment (MustafaNawzat) [reply src/App.js:14]: also, change `to` for the links to lower case.

**Comment by husseinTalal2** [comment]
it is not passing the tests because of treating warnings as errors in netlify, the warnings are `12:28:29 PM: ./src/components/FaqIntro/FaqIntro.jsx
12:28:29 PM:   Line 2:20:   'Button' is defined but never used  no-unused-vars
12:28:29 PM:   Line 25:13:  No duplicate props allowed          react/jsx-no-duplicate-props
12:28:29 PM: ./src/components/Questions/Qustions.jsx
12:28:29 PM:   Line 2:21:  'Accordion' is defined but never used              no-unused-vars
12:28:29 PM:   Line 2:32:  'Card' is defined but never used                   no-unused-vars
12:28:29 PM:   Line 8:21:  'setQuestions' is assigned a value but never used  no-unused-vars`
should I fix the errors or not ? @MustafaNawzat

**Comment by MustafaNawzat** [comment]
@husseinTalal2, just merged two pages could you pull then resolve the conflicts and make sure the route work for all merged pages.

**Pull request merged**